### ISSUE TITLE
Traceback error in bleachbit\GUI.pyo when cleaning is done on Windows

### ISSUE THREAD

**Issue by ineuw** [bleachbit/bleachbit#727]
In Window 10, I got a couple of messages but they didn't affect BB's function. 
The first was:  `exception when getting special folder path for pictures` because I removed the Microsoft family of default folders. (Documents, Pictures, etc.)
When cleanup completed, I got the following messages, so I thought that they may be of interest: 
```
Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 315, in run
  File "bleachbit\GUI.pyo", line 701, in worker_done
  File "gi\module.pyo", line 139, in __getattr_
```

**Comment by az0** [comment]
It seems the two errors are not related. 

Do you think the "exception when getting special folder path for pictures" error should be quietly ignored?

Do you have the option enabled to close BleachBit when it's done cleaning? You could try enabling or disabling that option, to check if it affects the error. That's what the GUI.py line 701 is about, though it's not obvious to me why an error would happen there.

**Comment by ineuw** [comment]
Hi and apologize for the delayed reply, but I needed time to implement a
process of elimination to find the source of the errors. The "pictures"
folder error message, when opening BB, was fixed by changing the user
permission.

The three .py files reported at the end of a BB cleanup event are triggered
by an acknowledged Windows 10 v1903 problem by a KB update file which
causes the "mcc.exe" to constantly crash. So please don't bother with
these. I am exploring possible fixes to this issue and posted about it
recently in the Windows 10 forum.

I tested Bleachbit by un-selecting every selected option, then closed and a
few times even rebooted before opening Bleachbit.The .py files report
disappear most of the time, but not always.

Take care

Imre

**Comment by ineuw** [comment]
The `exception when getting special folder path for pictures` was resolved by changing the permissions.

The above three file errors were generated selections in the Deep Scan options. Selecting any one of the four options will trigger the error message.

**Comment by assarbad** [comment]
I am getting the same error but I did _not_ remove these default folders.

This is how it looks.

```
Wiping free disk space erases remnants of files that were deleted without shredding. It does not free up space.
Disk space to be recovered: 6.4GiB
Files to be deleted: 7305
Special operations: 82
Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 315, in run
  File "bleachbit\GUI.pyo", line 701, in worker_done
  File "gi\module.pyo", line 139, in __getattr__
```

I'm seeing this on Windows 10 (1909) with BleachBit 3.0.

The folder permissions on the `Pictures` folder are inherited and provide full access.

**Comment by ediowar** [comment]
Windows 8 SL 64 bit, build 3.0.1.1447
`Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 317, in run
  File "bleachbit\GUI.pyo", line 715, in worker_done
  File "gi\module.pyo", line 139, in __getattr__`

Ubuntu 16.04.6 LTS, running from latest source code https://github.com/bleachbit/bleachbit/commit/82395794f191c58d509497eebcec4131c8e41565
`Traceback (most recent call last):
  File "/home/dzmitry/Downloads/bleachbit-master/bleachbit/Worker.py", line 317, in run
    self.ui.worker_done(self, self.really_delete)
  File "/home/dzmitry/Downloads/bleachbit-master/bleachbit/GUI.py", line 720, in worker_done
    notify.set_hint("desktop-entry", "bleachbit")
TypeError: argument value: Expected GLib.Variant, but got str
`

**Comment by az0** [comment]
On Windows 10, the error `  File "bleachbit\GUI.pyo", line 715, in worker_done` in BleachBit 3.2.0 and a similar error in BleachBit 3.0 happens when BleachBit tries to show a pop-up notification to alert the user that cleaning is done. It is triggered when cleaning takes longer than 10 seconds or when the BleachBit application window is not focused. The error is harmless because it's just a notification.

There seems to be several errors in this issue ticket, so I will change the issue title for clarity this is about the one issue. Then,  I will look more into it. If you have a different error that concerns you, please file a new ticket.

**Comment by az0** [comment]
Please test the fix in build 3.9.0.1534, which will be available soon at the[ BleachBit CI](https://ci.bleachbit.org/) site. 

To reproduce the bug, do a preview or scan that either (1) takes longer than 10 seconds or (2) finishes when the BleachBit window is not focused (e.g., minimized).

**Comment by ineuw** [comment]
For me, this issue disappeared with BB 3.20, in Linux and Windows 10

**Comment by az0** [comment]
The digitally-signed Windows installer is now available from the page [BleachBit 3.9.0 beta](https://www.bleachbit.org/news/bleachbit-390-beta)

**Comment by willbprog127** [comment]
Greetings,
I am receiving this error on 3.2.0 on Windows 10 Pro, on standard as well as admin type accounts after the cleaning is performed.  The cleaning takes about 1 minute and definitely not longer than 3 minutes.  The app is focused the entire time.

```
Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 317, in run
  File "bleachbit\GUI.pyo", line 715, in worker_done
  File "gi\module.pyo", line 139, in __getattr__
```
I have the output of the cleaning as well as the debug messages enabled, if needed.
Is the BleachBit 3.9.0 beta the solution to this issue?

Thanks! :+1:

**Comment by ineuw** [comment]
Check if you have access permission as an admin.  But, it's quicker to
download and upgrade to BB 3.9.0. It works for me well. No errors.

**Comment by az0** [comment]
This is the latest package available for testing:[ BleachBit 3.9.2 beta](https://www.bleachbit.org/news/bleachbit-392-beta)
